src/components/widgets/WidgetAdminEdit.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <router-link
+    v-if="hasEditRights"
+    class="widgetAdminEdit btn"
+    v-tooltip="adminLink.tooltip"
+    :to="adminLink.path"
+  >
+    <icon-edit class="widgetAdminEdit__actionIcon"/>
+    {{ adminLink.content }}
+    <icon-chevron-right class="widgetAdminEdit__chevronRightIcon" />
+  </router-link>
+</template>
+
+<script>
+import { mapActions, mapGetters, mapState } from 'vuex';
+
+export default {
+  name: 'WidgetAdminEdit',
+
+  components: {
+    IconEdit: () => import('@/components/IconEdit.vue'),
+    IconChevronRight: () => import('@/components/IconChevronRight.vue')
+  },
+
+  data: () => ({
+    disabled: false,
+  }),
+
+  computed: {
+    ...mapState([
+      'activePeriod',
+      'activeItem',
+      'activeObjective',
+      'activeKeyResult',
+    ]),
+    ...mapGetters(['hasEditRights']),
+    itemHomeAdminLink() {
+      return this.getAdminLink();
+    },
+    objectiveHomeAdminLink() {
+      return this.getAdminLink({
+        tab: 'okr',
+        type: 'objective',
+        id: this.activeObjective?.id,
+      });
+    },
+    keyResultHomeAdminLink() {
+      return this.getAdminLink({
+        tab: 'okr',
+        type: 'keyResult',
+        id: this.activeKeyResult?.id,
+      });
+    },
+    kpiHomeAdminLink() {
+      return this.getAdminLink({
+        tab: 'kpi' 
```

Comment:
Her er det trailing space (kan være kjekt og stille inn editoren til å slette trailing space automatisk ved lagring).

Vi bruker også trailing commas i for eksempel objekter som her. En fordel med det er at hvis noen skal legge til et nytt element her senere, så slipper de å endre linja du la til og forpurre historikken unødig. :)